dfget/core/downloader/power_client.go

```diff
@@ -71,13 +71,15 @@ func (pc *PowerClient) Run() (err error) {
 	}
 
 	// send download request
-	url := fmt.Sprintf("http://%s:%d%s", dstIP, peerPort, pc.pieceTask.Path)
-	headers := make(map[string]string)
-	headers["Range"] = "bytes=" + pc.pieceTask.Range
-	headers["pieceNum"] = strconv.Itoa(pc.pieceTask.PieceNum)
-	headers["pieceSize"] = fmt.Sprint(pc.pieceTask.PieceSize)
+	req := &api.DownloadRequest{
+		Path:       pc.pieceTask.Path,
+		PieceRange: config.StrBytes + "=" + pc.pieceTask.Range,
```

Comment:
construct `PieceRange` in download implementation

Reply:
Done.